### DISCUSSION TITLE
How to reassociate old users with new roles/permissions update

### DISCUSSION THREAD

**Discussion by ndethi** [spatie/laravel-permission#2271] [Q&A] [answered]
I dropped all roles and permissions tables created by this package from the db. I did a migration to restore them, but now the users permissions and roles attributes are empty. How would I re-associate them to the existing users?

**Comment by parallels999** [comment, marked as answer]
Manually or on a seeder/migration

**Reply by ndethi** [reply]
Thank you, did it via a seeder after setting up the roles and permissions.
